docs/content/en/docs/tasks/packages/metrics-server/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Metrics Server"
+linkTitle: "Add Metrics Server"
+weight: 13
+date: 2022-10-20
+description: >
+  Install/upgrade/uninstall Metrics Server
+---
+
+{{< content "../prereq.md" >}}
+
+
+## Install
+
+<!-- this content needs to be indented so the numbers are automatically incremented -->
+1. Generate the package configuration
+   ```bash
+   eksctl anywhere generate package metrics-server --cluster clusterName > metrics-server.yaml
+   ```
+
+1. Add the desired configuration to `metrics-server.yaml`
+
+   Please see [complete configuration options]({{< relref "../../../reference/packagespec/metrics-server" >}}) for all configuration options and their default values.
+
+    Example package file configuring a cluster autoscaler package to run on a management cluster.
+    ```yaml
+    apiVersion: packages.eks.amazonaws.com/v1alpha1
+    kind: Package
+    metadata:
+      name: metrics-server
+      namespace: eksa-packages-<cluster-name>
+    spec:
+      packageName: metrics-server
+      targetNamespace: <namespace-to-install-component>
+      config: |-
+        args:
+          - "--kubelet-insecure-tls"
+    ```
+
+
+2. Install Cluster Autoscaler
```

Comment:
In theory these should generate proper number list if you start them all with one, but this looks wrong either way. Maybe the double space here messed it up?

Reply:
Verified the auto-increment is working. I think we have the same issue in autoscaler and emissary. I'll touch both of them up.

Reply:
The auto-incrementing actually still seemed to work without changing everything to `1`. Regardless, I updated autoscaler and emissary to all use `1` now.

https://github.com/aws/eks-anywhere/pull/3769